packages/app/templates/ui-extensions/projects/pos_ui/src/index.liquid

```diff
@@ -1,3 +1,24 @@
+{%- if flavor == "react" -%}
+import React from 'react';
+import {render, useExtensionApi, extend, Tile, Text} from '@shopify/retail-ui-extensions-react';
+
+const Tile = () => {
+  const api = useExtensionApi();
+
+  return(
+    <>
```

Comment:
Yeah, using a fragment when there is only one child doesn't seem worthwhile.  I wonder if it's because we expect users to add multiple elements.

@dmhenry @samiulmonir06 do you have any context here?